src/Utils/Tempfile/TemporaryFile.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * This file is a part of NeklandTools package.
+ *
+ * (c) Nekland <github@nekland.fr>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+
+namespace Nekland\Utils\Tempfile;
+
+use Nekland\Utils\Exception\RuntimeException;
+use Nekland\Utils\Tempfile\Exception\CannotCreateFileException;
+use Nekland\Utils\Tempfile\Exception\ImpossibleToUpdateFileException;
+use Nekland\Utils\Tempfile\Exception\TemporaryFileAlreadyRemovedException;
+
+class TemporaryFile
+{
+    /**
+     * Tmp dir path. Do not use this var directly, uses `getTmpDirectory`.
```

Comment:
`Tmp dir path. Do not use this var directly, use `getTmpDirectory` instead.`

Is this comment mandatory ? The property is private ?

Reply:
Hum. This was an indication for a potential contributor. I can stay here I guess.